src/sagas/index.js

```diff
@@ -1,3 +1,4 @@
+import settingsSagas from './options'
```

Comment:
Nit-pick: Follow the same naming pattern as the other sagas and move the import down a few lines to be with them....

```suggestion
import sagasOptions from './options'
```